recipe/meta.yaml

```diff
@@ -51,9 +53,12 @@ test:
 about:
   home: https://github.com/Azure/azure-sdk-for-python
   summary: Microsoft Azure Machine Learning Client Library for Python
-  dev_url: https://github.com/Azure/azure-sdk-python
+  description: Microsoft Azure Machine Learning Client Library for Python
   license: Apache-2.0
+  license_family: Apache
```

Comment:
License could be MIT:
https://github.com/Azure/azure-sdk-for-python/blob/main/conda/conda-recipes/azure-ai-ml/meta.yaml#L72
https://github.com/Azure/azure-sdk-for-python/blob/main/LICENSE

It's not very clear.

Reply:
Indeed. I think it's MIT. https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/setup.py#L39